meetings/2015-03-09/minutes.md

```diff
@@ -0,0 +1,144 @@
+**SH:** We have a plan to put together this foundation proposal. io.js proposal was put together assuming TC would create documents, but Linux Foundation is handling.  
+**JZ:** That’s correct. The docs are created, but reviewed and edited by stakeholders. Good question regarding AB. At some point business decisions are made by Board of Directors and technical decisions are made by TC. The Foundation has rights to marks, but TC owns API compatibility.  
+**TM:** I’ve experienced that they last a little while in parallel (BoD and AB) and eventually the AB goes away. Usually a community group provides feedback to board.  
+**JZ:** To be clear, my idea is that the AB structure will dissolve. There will be alternate places where everyone participates in the new structure.  
+**BB:** Totally agreed. Could be a group focused on vendors and TC would go to them for understanding their needs.  
+
+
+
+
+#### Status from Linux Foundation (MD)
+**MD:** Reaching out to contacts to setup call for discussing drafts. Sent out drafts of bylaws. Working on technical discussions with BB and TF and have had a few calls. Hopefully will carry in to TC discussion. Drafted a blog post for communicating progress to community.  
+
+#### Node.js/io.js Reconciliation (BB)
+**BB:** There is this feeling that the stakeholders (io.js and Node.js) should get together in a room and hash things out. The io.js team didn’t really feel like it - would prefer to do it online, so a draft was written and posted for community feedback. Hopefully helps everyone understand the io.js community position. I’ve been working with MD and TF to find a compromise, but that isn’t going anywhere mostly because it’s not clear what the Node.js group wants. TF will send proposal after this meeting, but I’m concerned that I’m not negotiating with a “Node.js” group, and instead with only TF. Looking for input from people on this call as to opinions on io.js proposal.  
+**TF:** To clarify, there is a draft that will be circulated after this meeting. The Node.js team will schedule a call this week to discuss and draft will be sent out. Working on creating Technical Governance for foundation and creating a solid path forward for Node.  
+**BB:** May I suggest that the proposal goes on the internet somewhere.  
+**GR:** Who is on the Node.js committee?  
+**BB:** Participants from Joyent, Microsoft, and IBM. [Additional names provided, but not recorded.]  
+**GR:** Would love to see progress. I think io.js proposal is a good starting point and would like to understand what is on the other side. Lets make sure that there’s a good timeline in mind.  
+**TF:** Draft will be sent around after this meeting.  
+**DS:** Any thoughts on current io.js proposal?  
+**DC:** Some items WRT trademarks are something I haven’t seen before. For example, TC doesn’t typically own or manage trademark. Board manages that and issues resolved through discussion.  
```

Comment:
Agreed. Perhaps the conversation re: trademarks was misinterpreted and/or misconstrued.
The proposal is pretty clear/was clarified that the TC would not handle trademark issues.


A few select references:
https://github.com/iojs/io.js/issues/978#issuecomment-76322525
https://github.com/iojs/io.js/issues/978#issuecomment-76328274
https://github.com/node-forward/discussions/issues/31